src/main/java/com/intel/genomicsdb/GenomicsDBFeatureReader.java

```diff
@@ -20,26 +20,17 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-package genomicsdb;
+package com.intel.genomicsdb;
 
-import htsjdk.tribble.Feature;
-import htsjdk.tribble.FeatureReader;
-import htsjdk.tribble.FeatureCodec;
-import htsjdk.tribble.FeatureCodecHeader;
-import htsjdk.tribble.CloseableTribbleIterator;
-import htsjdk.tribble.readers.PositionalBufferedStream;
-import htsjdk.variant.bcf2.BCF2Codec;
-import htsjdk.variant.vcf.VCFHeader;
+import htsjdk.tribble.*;
 import htsjdk.variant.vcf.VCFContigHeaderLine;
+import htsjdk.variant.vcf.VCFHeader;
 
-import java.io.InputStream;
-import java.io.IOException;
 import java.io.File;
-import java.io.ByteArrayInputStream;
 import java.io.FileWriter;
-import java.util.List;
+import java.io.IOException;
 import java.util.ArrayList;
-import java.lang.RuntimeException;
+import java.util.List;
```

Comment:
ArrayList is still there. RuntimeException seems to be implicitly imported. I see that the code finds the definition both at compiler time and runtime